Wikipedia/Code/ClassNibView.swift

```diff
@@ -0,0 +1,9 @@
+
+class ClassNibView: UIView {
```

Comment:
makes it easier to use custom xib view classes in another xib or storyboard. makes it easier to break things up